### ISSUE TITLE
Residence (and it's child types)

### ISSUE THREAD

**Issue by gmackenz** [schemaorg/schemaorg#1270]
[Residence](http://pending.schema.org/Residence) currently has ApartmentComplex, GatedResidenceCommunity, & SingleFamilyResidence as child types (is there a granularity mismatch for these child types?) and there is some concern about how this fits into the LodgingBusiness/Accommodation model.
- One initial idea is to introduce a new type ResidenceComplex for the AC & GRC child types.

Previous related discussion on [issue#915](https://github.com/schemaorg/schemaorg/issues/915).

**Comment by danbri** [comment]
Thanks @gmackenz. Let's also note that the definition of Residence, "The place where a person lives." is somewhat lacking. The use of "The" is awkward to say the least; a place currently empty for rent or sale can still be residential.  Maybe something like "A place whose primary purpose is residential"?

**Comment by thadguidry** [comment]
@danbri @gmackenz Well...if the type was really meant to have the concept of "Residential" ... :)  Then we should rename it to that and clean up the definition.

**Comment by danbri** [comment]
@thadguidry it's more just bad wording. If somewhere is a residence, we don't want to say that it stops being one just because it is temporarily unoccupied.

**Comment by thadguidry** [comment]
@danbri yeah, I know...hence my smiley face.  Yes agree, a little tweak on the definition as you proposed and its good.

**Comment by thadguidry** [comment]
@danbri (reiterating in case you missed the updated comment)
BTW, what classifies a rename of a type ? stakeholder agreement ?

**Comment by danbri** [comment]
I missed the updated comment, indeed.

Generally we try to avoid renaming types, properties or for that matter enumerated values. It's more art than science. We'll be less keen to change terms that are in widespread use. We'll sometimes change term names for improved usability and coherence but we try to fight the urge to perfect everything. Beyond that, http://schema.org/docs/howwework.html probably covers it.

**Comment by thadguidry** [comment]
@danbri OK, thanks, yes, thats the idea I recall.  Speaking of which, I really dislike "byArtist"...which should be changed to "performer".  Anyways, thanks for the howwework pointer.

**Comment by danbri** [comment]
re "byArtist", I've just filed #1271.

**Comment by DaveyJake** [comment]
Why is `SingleFamilyResidence` found under `Thing > Place > Accommodation` when its description refers to it as a "residence type" on the page?  Is this just a mistake or is it also included under `Thing > Place > Residence`?
